app/models/form_handler.rb

```diff
@@ -7,29 +7,50 @@ def initialize
   end
 
   def get_form(form)
-    @forms[form]
+    @forms[form]["form"] if @forms[form].present?
   end
 
-  def current_form
-    forms[forms.keys.max_by(&:to_i)]
+  def current_lettings_form
+    forms["current_lettings"]["form"]
+  end
+
+  def current_sales_form
+    forms["current_sales"]["form"]
   end
 
   def sales_forms
     sales_sections = [] # Add section classes here e.g. Form::Sales::Property::Sections::PropertyInformation
-    { "2022_2023_sales" => Form.new(nil, "2022_2023_sales", sales_sections, "sales") }
+    current_form = Form.new(nil, current_collection_start_year, sales_sections, "sales")
+    previous_form = Form.new(nil, current_collection_start_year - 1, sales_sections, "sales")
+    { "current_sales" => { "form" => current_form, "type" => "sales", "start_year" => current_form.start_date.year },
+      "previous_sales" => { "form" => previous_form, "type" => "sales", "start_year" => previous_form.start_date.year } }
   end
 
   def lettings_forms
     forms = {}
     directories.each do |directory|
       Dir.glob("#{directory}/*.json").each do |form_path|
-        form_name = File.basename(form_path, ".json")
-        forms[form_name] = Form.new(form_path, form_name)
+        form = Form.new(form_path)
+        lettings_form_definition = { "form" => form, "type" => "lettings", "start_year" => form.start_date.year }
+
+        form_to_set = form_name_from_start_year(form.start_date.year, "lettings")
+        forms[form_to_set] = lettings_form_definition if forms[form_to_set].blank?
       end
     end
     forms
   end
 
+  def current_collection_start_year
```

Comment:
This maybe justifies it's own unit test? Unless there is one and I've missed it...